pulldown-cmark/src/lib.rs

```diff
@@ -224,6 +282,8 @@ pub enum Tag<'a> {
     },
 
     /// A metadata block.
+==== BASE ====
```

Comment:
Wrong doc comment here.

Reply:
ups, not sure how that happened, fixed it.